README.md

```diff
@@ -116,7 +116,9 @@ For more extensive guides and technical information please refer to our [Online
 ![screenshot](screenshots/screenshot8.png)
 
 # Installing Windows
-DOS Wasm X supports installing Windows 95 or Windows 98 using your own copy of Windows. Simply drag and drop the ISO onto the startup page. DOS Wasm X will detect the Windows CD and begin the installation process. If you choose to Install Windows 95 you may get the error below. Simply click OK and then cancel when it asks you for the Path to the CD. This will allow you to continue with the installation. The reason for this error is because at this stage of the process the CD drivers have not yet been loaded. However after restarting Windows it will detect the CD Drive and finish installing the drivers successfuly. Always remember to shut down windows in the guest OS before exiting the page. This will automatically save your hard drive changes to the browser and prevent scandisk from running the next time you boot into Windows.
+DOS Wasm X supports installing Windows 3.x, Windows 9x or Windows NT using your own copy of Windows. Simply drag and drop the ISO onto the startup page. DOS Wasm X will detect the Windows CD and begin the installation process. If you choose to Install Windows 95 you may get the error below. Simply click OK and then cancel when it asks you for the Path to the CD. This will allow you to continue with the installation. The reason for this error is because at this stage of the process the CD drivers have not yet been loaded. However after restarting Windows it will detect the CD Drive and finish installing the drivers successfuly. Always remember to shut down windows in the guest OS before exiting the page. This will automatically save your hard drive changes to the browser and prevent scandisk from running the next time you boot into Windows.
+
+If you try to install Windows NT. Make sure you install MS-DOS version 6.22 first, after that load your own copy of Windows NT and run this command "D:" (if you the DOS is already mount the copy of Windows then skip to the next step) then type "cd i386". After that type "winnt.exe /b" (NT 3.1 until NT 4.0) or "winnt.exe" (2000/NT 5.0) and press enter. Then when you have a SmartDrive error just ignore that and press enter. Then after the first phase setup is complete, press enter to reboot. After that continue the setup. It is recommended to keep the hard drive format is FAT or you cannot boot it.

```

Comment:
I think this type of information is better suited for the detailed docs here at the bottom:

https://github.com/nbarkhina/DosWasmX/blob/master/docs/README.md

if you want you can even create a dedicated windows-nt.md page where we can add information to over time.